transport-native-epoll/src/main/java/io/netty/channel/epoll/AbstractEpollChannel.java

```diff
@@ -59,6 +60,14 @@
         this.active = active;
     }
 
+    static boolean isActive(Socket fd) {
```

Comment:
consider an alternative name for this as in isolation it is not used to determine if a socket is active or not, but only when the socket is initialized. maybe something like:

```java
boolean isSoErrorZero(Socket fd) {
..
  return fd.getSoError() == 0;
..
}
```